drivers/hmis/app/graphql/mutations/ce/assign_referral_participants.rb

```diff
@@ -0,0 +1,48 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Mutations
+  class Ce::AssignReferralParticipants < CleanBaseMutation
```

Comment:
Just double checking, we need to use a bulk operation here and not CRUD mutations on the participant? Maybe CRUD doesn't align well with the UI.

Reply:
The UI is not finalized yet, but should look something like this (screenshot from figma mocks):

![image](https://github.com/user-attachments/assets/a9562b51-4070-4d6e-a128-74f106f8f267)